lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -44,9 +44,7 @@
 )
 from ccx_keys.locator import CCXLocator
 
-from ..models import (
-    CustomCourseForEdX,
-)
+from ccx.models import CustomCourseForEdX
 from ..overrides import get_override_for_ccx, override_field_for_ccx
```

Comment:
This is what I meant about not changing the adjacent imports.  I can make them the same if you would prefer.